samples/Mvc.Client/Startup.cs

```diff
@@ -12,7 +12,9 @@
 using Microsoft.Framework.Logging;
 
 namespace Mvc.Client {
+
```

Comment:
its the default behavior for code maid, since I I'm not a huge fan of same line curly brace, I just write things normally and I have a settings file for this repo to (hopefully) satisfy you all.....